lib/training/wiki_training_loader.rb

```diff
@@ -91,18 +91,25 @@ def training_content_and_translations(content:, base_page:, wikitext:)
     full_content
   end
 
-  # Gets a list of page titles linked from the base page
-  def wiki_source_pages
-    # To handle more than 500 pages linked from the source page,
-    # we'll need to update this to use 'continue'.
-    query_params = { prop: 'links', titles: @wiki_base_page, pllimit: 500 }
-    response = WikiApi.new(MetaWiki.new).query(query_params)
-    begin
-      response.data['pages'].values[0]['links'].map { |page| page['title'] }
-    rescue StandardError
-      raise InvalidWikiContentError, "could not get links from '#{@wiki_base_page}'"
+# Gets a list of page titles linked from the base page
+def wiki_source_pages
+  # To handle more than 500 pages linked from the source page,
+  # we'll need to update this to use 'continue'.
+  query_params = { prop: 'links', titles: @wiki_base_page, pllimit: 500 }
+  response = WikiApi.new(MetaWiki.new).query(query_params)
+  @query.merge! @continue unless @continue.nil?
```

Comment:
What is the sense of this line? If `@continue` is not previously defined then it will be always `nil`?

Reply:
@fillingtothemomo my previous comment was in relation to the following line:
`@query.merge! @continue unless @continue.nil?`

what I meant is that if `@continue` is not previously defined, then that line has no effect.

Reply:
@gabina I looked into continue implementation as in transclusion importer and I didnt see any explicit initialisation of @continue , can you elaborate on that?

Reply:
Could you please explain me what the following line does?
`@query.merge! @continue unless @continue.nil?`

Reply:
And what is the content of  `@continue`?

Reply:
It's nil currently.  I tried with ` @continue = response['continue'] ` as well so it wouldn't be null, but didn't succeed

Reply:
Ok, you have to fix that. What the answer looks like? That should help you understand why `@continue = response['continue']` doesn't work.